src/bulksnpgen.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+source activate bam-readcount
```

Comment:
There is a way to activate a conda environment for a single snakemake rule: https://snakemake.readthedocs.io/en/stable/snakefiles/deployment.html#combining-conda-package-management-with-containers

Can we use that instead of activating the environment in the shell script?

Reply:
In the original version of scsim, it uses source activate so I just follow the style. Sure, I can change to conda activate.